### DISCUSSION TITLE
Live Demo

### DISCUSSION THREAD

**Discussion by dwi110** [tomek199/RankMe#208] [Q&A]
Hey,
is there anywhere a live demo of RankMe, where you can see RankMe in action?

**Comment by tomek199** [comment]
Hi @dwi110 
Unfortunately the application is not deployed yet as it's not in "ready to use" state. 
It's just a hobby project which is still under development, so it might take a time to release stable version. When it will be ready to use, I will think about some deployment, but I don't know when it's going to happen :smile:

**Comment by dwi110** [comment]
Thank you @tomek199  for the fast reply.
Iam very interested in this project as i am from "wecreatefootball" - we are players of the sport flag football and we are trying to create an european wide ranking for all teams. as a our jounrey began in the last monthts, we saw what a struggle a frontend will be (and backend). there are not many gliccko based projects around. whats missing for a ready to use state? at the moment we use the chess excel spreadsheet that can be found on the glicko homepage.

if you need example data - we already gathered a bunch of leagues and scores from international tournaments. we would love to help you at your project. just hit me up if we can help you anywhere

**Reply by tomek199** [reply]
Yes, I'm aware that there are not so many applications to handle such glicko rankings, that's why I started to build this one :) 
I would say that very basic functionality is cover - creating new leagues, adding players and play games, BUT there is no authentication/authorization mechanism implemented at all. This is a crucial part that has to be done as a first glance. Let me think how to handle that.

**Reply by dwi110** [reply]
Nice, thats still sounding good. We will experiment with RankMe this week. Already the basic funtions will help us hopefully a lot, even if we can use it just locally
